homeassistant/components/smartthings/manifest.json

```diff
@@ -1,4 +1,5 @@
 {
+  "version": "1.0.0",
```

Comment:
This is not correct, versions should not be set on core integrations.

```suggestion
```

Reply:
ok, will remove. It was needed when testing as a custom component.